src/_posts/databases/postgresql/2000-01-01-managing.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Managing Your Scalingo for PostgreSQL® Addon
+nav: Managing
+modified_at: 2023-12-01 00:00:00
+tags: databases postgresql addon
+index: 5
+---
+
+
+## Scaling (Changing Plan)
+
+You can change your database plan whenever you want. This operation happens
+instantly, no manual input is required. When you change the plan, the instances
+of your database are stopped and then restarted with the characteristics of the
+chosen plan. According to the type of plan you are originally using, the impact
+on your application differs:
+- **Starter Plans**: Since these plans are not highly available, expect around
+  2 minutes of unavailability during which the scale operations are conducted.
```

Comment:
question: I find the use of the words "2 minutes" misleading. From what I remember, it actually highly depends on the amount of data. For a change of plan from Starter 8G to 16G, I'm pretty sure it takes much more time, isn't it?

Reply:
Actually, I don't know. But that's what we state in the current documentation: https://doc.scalingo.com/databases/postgresql/start#changing-plans
I'll rephrase it :smiley: